lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -259,6 +262,18 @@ def transform_gulf_war(gulf_war1990, gulf_war2001)
         gulf_war_hazard_service
       end
 
+      def transform_herbicide(herbicide)
+        filtered_results_herbicide = herbicide&.filter { |k| k != 'notsure' }
+        herbicide_value = filtered_results_herbicide&.values&.any?(&:present?) &&
+                          !none_of_these(filtered_results_herbicide)
+
+        herbicide_service = Requests::HerbicideHazardService.new
+        herbicide_service.served_in_herbicide_hazard_locations =
+          herbicide_value || herbicide['otherHerbicideLocations'].present? ? 'YES' : 'NO'
```

Comment:
what happens if we have something like this for the `otherHerbicideLocations`?
```
"otherHerbicideLocations": {
          "description": "",
          "startDate": "",
          "endDate": ""
        },
```
or 
```
"otherHerbicideLocations": {
          "description": null,
          "startDate": null,
          "endDate": null
        },
```
?
🤔

Reply:
we're now accounting for this scenario